### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by curioswati-zz**
@pulkitvaishnav, these are few fixes and updates. Please see them once.

### CODE REVIEW

File: login_utils.py [hunk 1]
```diff
@@ -136,13 +156,20 @@ def normal_login(session, username, password, continue_url):
...
 
+    # if the page did not have the form it won't have payload, that shows the response page has
+    # changed or the request was not appropriate.
+    if not payload:
+        file_name = utils.log_error("normal login", form_html.text)
+        error = 500
+        return form_html, error, session
+
     # add email, password and target url in payload
     payload['Email'] = username
```
Comment (pulkitvaishnav) [inline login_utils.py:174]: Asking just for curiosity you are taking some character some key e.g. Email & Passwd of payload dictionary in Capitalize way and  'continue' in lower case. Any reason?

File: utils.py [hunk 2]
```diff
@@ -155,3 +162,53 @@ def scrap_error(page):
         error = None
 
     return error
+
+
+def log_error(step, content):
+    '''
+    This function logs a page for error.
+    It is called whenever an unhandled exception will occur.
+    `step`: The step in login process, it cold be normal_login, step_two_login or select_alternate.
```
Comment (pulkitvaishnav) [inline utils.py]: I think you are going to write this  "it could be normal_login" in place of "it cold be normal_login,"

**Comment by pulkitvaishnav** [comment]
Rest LGTM merge it. If it working correctly.

File: login_utils.py (hunk 1 above)
Comment (curioswati-zz) [reply login_utils.py:174]: These are form parameters, the name of the form fields for them. So taking them as they are.

File: utils.py (hunk 2 above)
Comment (curioswati-zz) [reply utils.py]: :-P yes.

**Pull request merged**